packages/node_modules/@webex/internal-plugin-calendar/src/calendar.js

```diff
@@ -34,6 +37,32 @@ const Calendar = WebexPlugin.extend({
       qs: options
     })
       .then((res) => res.body.items);
+  },
+
+  /**
+   * Retrieves an array of meeting participants for the meeting id
+   * @param {String} id
+   * @returns {Promise} Resolves with an object of meeting participants
+   */
+  getParticipants(id) {
+    return window.squared.request({
+      method: 'GET',
+      service: 'calendar',
+      resource: `calendarEvents/${btoa(id)}/participants`
+    }).then((res) => res);
+  },
+
+  /**
+   * Retrieves a collection of meetings based on the request parameters
+   * @param {String} id
+   * @returns {Promise} Resolves with an object of meeting notes
+   */
+  getNotes(id) {
+    return window.squared.request({
+      method: 'GET',
+      service: 'calendar',
+      resource: `calendarEvents/${btoa(id)}/notes`
+    }).then((res) => res);
```

Comment:
This `.then((res) => res);` is redundant in this case, please remove this `.then()` statement.